Formula/llvm.rb

```diff
@@ -141,6 +142,13 @@ def install
       system "cmake", "--build", ".", "--target", "install-xcode-toolchain" if MacOS::Xcode.installed?
     end
 
+    on_macos do
+      if build.stable? && !(lib/"libLLVM-#{version.major}.dylib").exist?
```

Comment:
When does/doesn't this exist?

Reply:
The last time I built from HEAD it installed a `libLLVM-13git` library, so I assume it exists in HEAD builds. Verifying this now is difficult because building from HEAD seems to fail at the moment, plus it takes ages.

I'm not aware of any circumstances for it to exist in a stable build. I just put it there because it was going to be wrapped in a conditional anyway (`build.stable?`), so may as well add an extra guard to avoid overwriting this library if/when this bug does get fixed.

We can drop the existence check if you prefer.

Reply:
Yeh, dropping existence check makes sense to me 👍🏻. Lets make this error out if it ever starts to exist so we know to remove this.

Reply:
Done. Also, added a test to help us more quickly detect breakage from this.